File: Patterns/python/pattern14.py
# n = 3

# A
# B B
# C C C

n = int(input())
row = 1
startChar = 65
while row <= n:
    col = 1
    while col <= row:
        print(chr(startChar), end="")
        col += 1
    row += 1
    startChar += 1
    print("")

File: Patterns/js/pattern14.js
/*
n = 3

A
B B
C C C

*/

function pattern(n) {
  let ans = "";
  let row = 1;
  let startingCharacter = 65;
  while (row <= n) {
    let col = 1;

    while (col <= row) {
      ans += String.fromCharCode(startingCharacter) + " ";
      col++;
    }

    startingCharacter++;
    row++;
    ans += "\n";
  }

  console.log(ans);
}

pattern(3);
pattern(5);
